internal/server/statvar/golden/search_statvar/count_person.json

```diff
@@ -3899,106 +3999,6 @@
     {
       "name": "Population: 25 - 34 Years, Native Born Outside The United States",
       "dcid": "Count_Person_25To34Years_NativeBornOutsideTheUnitedStates"
-    },
```

Comment:
Not sure if this truncation is expected..

Reply:
I don't understand this.  There was no change to these imports (subject table S270*). My guess might be that these somehow went into Uncategorized vertical, but they do show under Demographics.

Also, the first time I went to the link below, I got nothing...

https://autopush.datacommons.org/tools/statvar#Count_Person_25To34Years_NoHealthInsurance

Reply:
Got nothing, I mean like: https://autopush.datacommons.org/tools/statvar#foo

@shifucun  you might have some insights... what cache does this use? does it do merge?